PluginCode/src/main/java/smr/cs/ualberta/libcomp/DatabaseAccess.java

```diff
@@ -481,16 +484,67 @@ public void updateUserProfile(User userRecord) throws IOException {
 
         if (Integer.parseInt(userRecord.getCloudStore()) == 1) {
             sendUser(userRecord.getUserID(), obj.toString());
+
+            sendFeedbackToCloud(); // check if the user has local feedback stored previously, send them now.
```

Comment:
What's teh difference between `sendFeedback` above? and `sendFeedbackToCloud()`?

Reply:
sendFeedbackToCloud() will loop over all feedback entries and call sendFeedback() for each single entry
sendFeedback() sends a single json entry to smr

Reply:
So as you can see both actually send feedback to the cloud but their names don't distinguish that. Perhaps it should just be
`sendAllLocalFeedback`
and `sendFeedbackEntry`

or 
`sendFeedback`
and `sendFeedbackEntry`

something that indicates that one loops over everything while the other is a helper/subtask method to send a single entry